gensrc/thrift/Opcodes.thrift

```diff
@@ -86,6 +86,12 @@ enum TExprOpcode {
     BIT_SHIFT_LEFT,
     BIT_SHIFT_RIGHT,
     BIT_SHIFT_RIGHT_LOGICAL,
+    MATCH_TERM,
+    MATCH_FUZZY,
+    MATCH_WILDCARD,
+    MATCH_ALL,
+    MATCH_PHRASE,
+    MATCH_RANGE,
```

Comment:
Why do you introduce these Expr opcodes?
For now, starrocks doesn't depend on these opcodes. You can just use the function framework to support search operations.

Reply:
I will remove these, and can you show me an predicate function example code?

Reply:
I think `Like` will be a good case or `regexp_replace`

Reply:
Thanks you, I will implement various `MatchPredicate` by referring to the implementation of the `Like` in following PR.